app/lib/screens/study/dashboard/dashboard.dart

```diff
@@ -40,7 +41,9 @@ class _DashboardScreenState extends State<DashboardScreen>
   List<TaskInstance>? scheduleToday;
 
   bool get showNextDay =>
-      (kDebugMode || context.read<AppState>().isPreview) &&
+      (kDebugMode ||
+          context.read<AppState>().isPreview ||
+          context.read<AppState>().isDebugModeEnabled) &&
```

Comment:
Having different "debug" variables is also not optimal, can become confusing. Maybe we can combine all of them and use them in the whole application?
So we can have:

`debugMode = isPreview || kDebugMode`

And use `debugMode` everywhere instead of `kDebugMode`. What do you think?